### ISSUE TITLE
Wind Plot symbols not drawing correctly

### ISSUE THREAD

**Issue by BobBooth** [Esri/military-symbology#274]
The Wind Plot symbols do not draw correctly in 2525C - a single symbol draws, but the symbol should vary based on wind speed and direction.
METOC > Atmospheric > Winds > Wind plot

WAS-WP----P---- 
Wind Plot (METOC.AMPHC.WND.PLT)

![image](https://user-images.githubusercontent.com/3801392/44682411-4a267380-aa11-11e8-87a8-e4cde353eb42.png)

This is due to the fact that the various different symbols (shaft, barbs, half barbs, and pennants, as well as a central cloud cover circle) indicating different wind speeds and conditions share the same SIC. 2525C pages 806, 807)

We appear to have the same problem in our 2525D style:
![image](https://user-images.githubusercontent.com/3801392/44682745-40e9d680-aa12-11e8-8422-58aa5ba6bd1b.png)

**Comment by joebayles** [comment]
https://github.com/Esri/joint-military-symbology-xml/issues/182

TL;DR: Scroll to the bottom and click on the link to see the entire set of symbols. Tough part is making it work w/ attributes and not SIDCs.

**Comment by lfunkhouser** [comment]
This issue was moved to ArcGIS/military-symbology-styles#67
